build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.0'
+        classpath 'com.android.tools.build:gradle:2.1.2'
```

Comment:
When upgrading the Gradle version, also remember to upgrade the corresponding gradlew wrapper files. This is the command:

`./gradlew wrapper --gradle-version 2.12`

This will re-generate all necessary Gradle files.